### ISSUE TITLE
Restricted ad serving

### ISSUE THREAD

**Issue by peterel** [admob-plus/admob-plus#488]
**Describe the bug**
Just as I switched to this plugin, Admob Restricted my ads. My app had been running in same manner for years without this happening. Could be a huge coincidence of course but could there be anything in switching to this plugin that could have caused this? Flagged as "Site Behaviour: Navigation" (and in no way am I violating anything relating to that policy).

https://apps.apple.com/us/app/push-penguin/id1210906366

Many thanks in advance!

**Comment by krunalsk007** [comment]
can you share some code of ad ?

**Comment by peterel** [comment]
@krunalsk007 Thanks for your reply. 

Code is very much per doc I believe, see below. Can the implementation cause such a restriction from Google you believe? NOTE: Before below, I ask consent for "tracking".

On deviceReady:
`admobid.interstitial = 'ca-app-pub-X';
interstitial = new admob.InterstitialAd({
	adUnitId: admobid.interstitial,
    })

interstitial.on('load', (evt) => {
	// evt.ad
	console.log("ad is loaded");
})

interstitial.load();`

When Showing ad:
`//show ad
if(platform!="web") {
   interstitial.show();
}`

On admob.ad.dismis:
` if(platform!="web") interstitial.load();`

**Comment by DesignerApparelSales** [comment]
I also have ads serving limits from admob, not sure if it from this plugin tho, But I never have it before until a month after I switch to this plugin. I ended up deleted all my ads units and the limits went away,

**Comment by peterel** [comment]
Many thanks for your reply. So you just deleted and created new ones, and
all is now good?

Peter

**Comment by DesignerApparelSales** [comment]
Yeah, I deleted the ads unit, unpublish my app and waited a week. now the limits is gone. Now I have to install the new ads units and upload a new version. But beware, for whatever reason, I can't get this plugin to work any more with the new build. No ads are show. I think it have to be all the new updates and new sdk. I haven't update my app in a while. I'm in a process of debugging the problems right now. All of this is just a major hassle. Keep in mind, once you deleted all the ads units, you not going to make anymore money until you add a new ads unit. just test it first and do more research before you do anything. This was my last resort since my app stop making money for 2 months.

**Comment by peterel** [comment]
Got it, many thanks!

I was using another plugin earlier, and all was fine and well. Changed to
this one, indeed with a lot of hassle as I had not updated my app for some
time, and now these problems. Will give it some though and see if I delete
and re-add. Thanks again.

Peter

**Comment by DesignerApparelSales** [comment]
What is the name of the other plugin that you use? I might tried that plugin if this plugin give me ads limits in the future.

**Comment by mirfanazam** [comment]
I am also facing ad restriction while using this plugin. For sure, there is something with this plugin.

**Comment by DesignerApparelSales** [comment]
Just today , I got  Flagged as "Site Behaviour: Navigation" and got ads limited serving. Did you ever find a solution to your problem?

**Comment by DesignerApparelSales** [comment]
Nevermind, I removed my interstitial ads, and the problem is gone. I think alot of these ads serving limit problem is from google bug and not this plugin. This plugin is great. Thanks to the creator

**Comment by peterel** [comment]
Thanks for the info. But what do you mean by “I removed my interstitial
ads”? You removed and and added them again from admob (resulting in new ID)
and updated your app with the new admob-id:s?

**Comment by DesignerApparelSales** [comment]
Nah, I just removed them all togther because i don't need it and it is annoying af. I just have banner now. I think what happen is the interstitial ads pop up at a wrong place that is why you get "Site Behaviour: Navigation" or people accidently click into it. You can also go into admob account and into the add unit setting and set the ads to $100 eCPM floor so the intersitial ads don't show up. I didn't do that, I just removed it all together in my  code and resubmit my app.  I then click "fix" in my admob. They review it in a couple of hours and remove the ads limits serving. Now it is back to normal.

**Comment by ftheriault** [comment]
I just wanted to say that I have the same problem, using reward ads. If anyone knows a fix for this, please share with us.

**Comment by Carlosps** [comment]
One option would be start using this plugin [Applovin](https://www.npmjs.com/package/cordova-plugin-applovin-max).

This is an ad network provider just like Admob, you can also setup mediation so that you can also use (Face ads, Admob, Pangle...)

**Comment by EYALIN** [comment]
@ratson i think you can close this ticket
